src/dotnet/APIView/APIViewWeb/LeanModels/ChangeHistory.cs

```diff
@@ -12,11 +12,54 @@ public enum AICommentChangeAction
         Modified
     }
 
-    public class AICommentChangeHistoryModel
+    [JsonConverter(typeof(StringEnumConverter))]
+    public enum ReviewChangeAction
+    {
+        Created = 0,
+        Closed,
+        ReOpened,
+        Approved,
+        ApprovalReverted,
+        Deleted,
+        Undeleted
+    }
+
+    [JsonConverter(typeof(StringEnumConverter))]
+    public enum ReviewRevisionChangeAction
```

Comment:
nit: You can rename this to `RevisionChangeAction` to trim it down.

Reply:
Right. However, samples have Revisions too so I started using the full name just to differentiate the two.